New package: RogueWaveInfiniteNLS v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: RogueWaveInfiniteNLS
- Repository: https://github.com/bilman/RogueWaveInfiniteNLS.jl
- Created by: @bilman
- Version: v0.1.0
- Commit: 28f5c4b4d196cfdf2b38cfd487ac1c118394996d
- Reviewed by: @bilman
- Reference: https://github.com/bilman/RogueWaveInfiniteNLS.jl/issues/2
- Description: Julia package for computing the rogue waves of infinite order for the focusing nonlinear Schrödinger equation.
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
Initial release with the fixes.
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598d6f67527f9d940e91a1cfff195c568f39e2a4ee15983661e9e38b8798e6e13a891cc1c9c974838766384c48f297ce67f8fd081d34676b4aac77347cefb494c6896c6c3f120d712e6144860a4924cf5faf16114a689b11a3fdeeb150541c8fc70f39c5c61b694045e9d9e365ea1916195a8498e6d77bb6833c57c2cc25e30abd5 -->

goerz commented:
[noblock] Looks nice, and should be good to go as soon as you fix the compat issue.

I would point out that "Schrodinger" should have an umlaut ("Schrödinger"), both in the README and in the project description line. Especially with Julia, we live in a unicode world, so let's not promulgate these kinds of misspellings. For what it's worth, if you have to convert umlauts to the ASCII alphabet, the correct way of doing so is by appending an "e" to the vowel, so "Schroedinger" would be more correct (interesting tidbit: umlauts are the remnants of writing a small "e" on top of the preceding vowel)

bilman commented:
> [noblock] Looks nice, and should be good to go as soon as you fix the compat issue.
> 
> I would point out that "Schrodinger" should have an umlaut ("Schrödinger"), both in the README and in the project description line. Especially with Julia, we live in a unicode world, so let's not promulgate these kinds of misspellings. For what it's worth, if you have to convert umlauts to the ASCII alphabet, the correct way of doing so is by appending an "e" to the vowel, so "Schroedinger" would be more correct (interesting tidbit: umlauts are the remnants of writing a small "e" on top of the preceding vowel)

Thanks! Fixing them now.

JuliaRegistrator committed:
New package: RogueWaveInfiniteNLS v0.1.0

UUID: deca77a3-49e2-4fa7-ba04-95ade2bf29f1
Repo: https://github.com/bilman/RogueWaveInfiniteNLS.jl.git
Tree: cc50d2784b9f0325bc64ea0c08ea02eaeaa9c080

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

goerz commented:
[noblock] 👍 You’ll want to edit you comment to contain `[noblock]`

bilman commented:
> [noblock] 👍 You’ll want to edit you comment to contain `[noblock]`

[noblock] I think I successfully edited my initial comment (response to you) now by including [noblock] at the beginning... I am not sure what this serves for, though. My first time registering a package!

goerz commented:
[noblock] Yes, you're good to go. The package should automatically be merged after the three-day waiting period ends.

As the bot explained in https://github.com/JuliaRegistries/General/pull/110936#issuecomment-2225221802 (see the "Tip" at the bottom of the message), any comment that does not contain `[noblock]` will prevent the package from being merged.
